src/topology/algebra/open_subgroup.lean

```diff
@@ -131,29 +124,24 @@ def prod (U : open_subgroup G) (V : open_subgroup H) : open_subgroup (G × H) :=
 end
 
 @[to_additive]
-instance : partial_order (open_subgroup G) :=
-{ le := λ U V, ∀ ⦃x⦄, x ∈ U → x ∈ V,
-  .. partial_order.lift (coe : open_subgroup G → set G) coe_injective }
+instance : has_inf (open_subgroup G) :=
+⟨λ U V, ⟨U ⊓ V, U.is_open.inter V.is_open⟩⟩
+
+@[simp, norm_cast, to_additive] lemma coe_inf : (↑(U ⊓ V) : set G) = (U : set G) ∩ V := rfl
```

Comment:
Can you add the `sup` version of this lemma too?

Reply:
The `sup` version only works for coercion to `subgroup G`.